services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import MovieSession, Order, Ticket
+from django.db.models import QuerySet
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str = None,
+        date: str = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+
+    if date:
+        order.created_at = date
+        order.save()
+
+    for ticket in tickets:
+        Ticket.objects.create(
+            movie_session=MovieSession.objects.get(
+                id=ticket["movie_session"]
+            ),
+            order=order,
+            row=ticket["row"],
+            seat=ticket["seat"]
+        )
+
+
+def get_orders(username: str = None) -> QuerySet:

```

Comment:
Indicate a `QuerySet` of what instances you return in the return type annotation